src/auslib/web/public/swagger/api.yml

```diff
@@ -161,6 +165,7 @@ paths:
         - $ref: '#/parameters/avast'
         - $ref: '#/parameters/force'
         - $ref: '#/parameters/mig64'
+        - $ref: '#/parameters/pin'
```

Comment:
We talked about this elsewhere, but just noting here we don't actually need this param in endpoints other than `/update/6`.  It doesn't hurt elsewhere though, so I think it's fine to keep it.